AutomatorServer/ios/AutomatorServerUITests/PatrolAutomation.swift

```diff
@@ -178,65 +196,82 @@ class PatrolAutomation {
     }
   }
   
+  // MARK: Notifications
+  
+  func openNotifications() {
+    // TODO: Check if works on iPhones without notch
+    
+    runAction("opening notifications") {
+      let start = self.springboard.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.01))
+      let end = self.springboard.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.6))
```

Comment:
Oh boy, I dont know iOS, but isnt this gesture iOS-version-related? Is there really no method for doing it?

Reply:
It is, but this behavior seems to be consistent across the newest iOS versions.

> Is there really no method for doing it?

Unfortunately no. That's all we have at our disposal.